google-cloud-firestore/samples/acceptance/query_watch_test.rb

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper.rb"
+require_relative "../query_watch.rb"
+
+describe "Google Cloud Firestore API samples - Query Data" do
+  before :all do
+    @firestore_project = ENV["FIRESTORE_PROJECT"]
+  end
+
+  before do
+    @collection_path = random_name "cities"
+  end
+
+  after do
+    delete_collection_test collection_name: @collection_path, project_id: ENV["FIRESTORE_PROJECT"]
```

Comment:
Any reason you're not using `@firestore_project`?

Reply:
This was inherited from ruby-docs-samples.

Reply:
Fixed.